superset-frontend/spec/helpers/testing-library.tsx

```diff
@@ -64,11 +71,18 @@ export function createWrapper(options?: Options) {
     if (useRedux) {
       const mockStore =
         store ??
-        configureStore({
-          preloadedState: initialState || {},
-          devTools: false,
-          reducer: reducers || reducerIndex,
-        });
+        (!initialState
```

Comment:
Just a nit but you could invert the order and not negate the condition:

`initialState ? configureStore : storeWithApi`